cypress/SETUP.md

```diff
@@ -0,0 +1,35 @@
+### Pre-requisites
+
+This document assumes that you have valid read-only access to the **production** database, in order to extract the data of the e2e test sites.
+
+### Setup
+
+#### Github e2e tests
+
+In order to run the github e2e tests successfully, we need to perform the following one-off setup:
+
+1. go to the backend folder (assuming that both FE + BE are in the same folder, you can `cd ../isomercms-backend`)
+2. set `E2E_ISOMER_ID` to your desired value
+3. ensure that the `E2E_TEST_GH_TOKEN` has sufficient permissions (try running any test)
+
+#### Email e2e tests
+
+In order to run the email e2e tests successfully, we need to perform the following one-off setup:
+
```

Comment:
hmm could we use a script to automate this in the backend? the repos has a mapping to the `sites` so there will be a manual change of the site id to match local db, which isnt ideal

Reply:
good suggestion but probably not going to use a script - i'll be making this a migration with `upsert` so that it's idempotent as `staging + production` already have this